web_readonly_bypass/__openerp__.py

```diff
@@ -25,10 +25,6 @@
     "maintainer": "ACSONE SA/NV, Odoo Community Association (OCA)",
     "website": "http://www.acsone.eu",
     'category': 'Technical Settings',
-    'depends': [
-        'base',
-        'web',
-    ],
```

Comment:
I wonder if/how javascript unit-test are launch by travis? I've done a quick test adding a test that failed but I haven"t fiend any logs about: https://travis-ci.org/petrus-v/petrus-v/web/builds/83849923